### ISSUE TITLE
a11y: External/new window icons missing text equivalent

### ISSUE THREAD

**Issue by iamjolly** [18F/all_sorns#147]
## Description:
The external/new window icons on links are rendered using a CSS generated background image which does not have the ability to offer text alt with it like a foreground image would. This means that screen reader users who cannot see the icon do not get the same information as a sighted user does.

## Screenshots:
![screenshot of generated content icons with annotations](https://user-images.githubusercontent.com/1093423/105558841-d6b1ea80-5ccc-11eb-9713-57c419d695bb.png)
![screen shot of accessibility tree results of generated content icons](https://user-images.githubusercontent.com/1093423/105558846-d87bae00-5ccc-11eb-82df-7dd1d2975ce4.png)

## WCAG SC
[Success Criterion 1.1.1 Non-text Content (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content): All non-text content that is presented to the user has a text alternative that serves the equivalent purpose...

## Recommended fix:
Either of the following fixes would be appropriate for this issue:
- Use a foreground image of the icon (instead of CSS generated background image) with `alt` set to "opens in new window", or...
- Use visibly-hidden text within the link text (apply `.usa-sr-only` class to the `span`) to append "- opens in new window" to the link text, or...
- Remove `target="_blank"` and `.usa-link--external` class from the links.
